config/db/0000_initialize.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `dice_emails` (
+  `registered_email` varchar(255) NOT NULL,
+  UNIQUE KEY `registered_email` (`registered_email`)
+) DEFAULT CHARSET=latin1;
+
+CREATE TABLE `pending_validations` (
+  `email` varchar(255) NOT NULL,
+  `validation_key` char(32) NOT NULL,
+  `time_stamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `IP` int(10) unsigned NOT NULL,
```

Comment:
Are you going to create a 0001 file to remove this column?

Reply:
Most likely.  However, I'm probably going to submit a short-term fix that simply hardcodes `IP` to zero in the PHP code to get IPv6 working as quickly as possible without having to schedule downtime to migrate the DB.